### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by End-rey**
Refs #2943.

### CODE REVIEW

File: cmd/neofs-adm/internal/modules/morph/local_client.go [hunk 1]
```diff
@@ -212,13 +212,13 @@ func (l *localClient) CalculateNetworkFee(tx *transaction.Transaction) (int64, e
...
-		netFee += fee
+		newFee, sizeDelta := fee.Calculate(ef, verificationScript)
+		netFee += newFee
 		size += sizeDelta
 	}
 
-	fee := l.bc.FeePerByte()
-	netFee += int64(size) * fee
+	feePerByte := l.bc.FeePerByte()
+	netFee += int64(size) * feePerByte
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/morph/local_client.go:221]: used exectly once, so easier to drop the var

File: pkg/local_object_storage/engine/lock_test.go [hunk 2]
```diff
@@ -16,7 +16,6 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	objecttest "github.com/nspcc-dev/neofs-sdk-go/object/id/test"
-	oidtest "github.com/nspcc-dev/neofs-sdk-go/object/id/test"
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/engine/lock_test.go:19]: it is `oidtest` actually, `objecttest` was redundant

File: pkg/services/control/service_test.go [hunk 3]
```diff
@@ -101,6 +101,7 @@ func equalListShardResponseBodies(b1, b2 *control.ListShardsResponse_Body) bool
 
 	return true
 }
+
```
Comment (cthulhu-rider) [inline pkg/services/control/service_test.go:104]: `fmt` caused change, not lint

File: cmd/neofs-adm/internal/modules/config/config_test.go [hunk 4]
```diff
@@ -35,11 +35,11 @@ func TestGenerateConfigExample(t *testing.T) {
 	require.Equal(t, 100000000, v.GetInt("network.fee.withdraw"))
 
 	var i int
-	for i = 0; i < n; i++ {
+	for i = range n {
```
Comment (cthulhu-rider) [inline cmd/neofs-adm/internal/modules/config/config_test.go:38]: `:=` and drop `var i` declaration?

File: pkg/morph/timer/block_test.go [hunk 5]
```diff
@@ -208,7 +208,7 @@ func TestBlockTimer_TickSameHeight(t *testing.T) {
 	require.NoError(t, bt.Reset())
 
 	check := func(t *testing.T, h uint32, base, delta int) {
-		for i := 0; i < 2*int(blockDur); i++ {
+		for range 2 * int(blockDur) {
```
Comment (cthulhu-rider) [inline pkg/morph/timer/block_test.go:211]: cast to `int` is not needed now?

**Comment by roman-khimov** [comment]
> squash all commits into one

Please don't. They're nice this way, each commit fixing some linter of its own. Really easy to look through.

**Comment by cthulhu-rider** [comment]
> They're nice this way

> Really easy to look through.

too personal and doubtful. In opposite, to me, service commits make it more difficult to navigate foundational changes

**Comment by roman-khimov** [comment]
> too personal and doubtful

Each commit solves some problem of its own, no doubt it's better.

**Comment by cthulhu-rider** [comment]
> Each commit solves some problem of its own

having lint action failures -> no failures - this is the most reasonable atomic problem statement for this project to me. And 20 linter-related commits taking up the entire screen is just a distraction

finally, my point remains, but since the work of separation has been done, let it remain too

File: pkg/local_object_storage/metabase/delete_test.go [hunk 6]
```diff
@@ -61,10 +61,10 @@ func TestDB_Delete(t *testing.T) {
 	// check if they marked as already removed
 
 	ok, err := metaExists(db, object.AddressOf(child))
-	require.Error(t, apistatus.ObjectAlreadyRemoved{})
+	require.Error(t, err, apistatus.ObjectAlreadyRemoved{})
 	require.False(t, ok)
 
-	ok, err = metaExists(db, object.AddressOf(parent))
+	ok, _ = metaExists(db, object.AddressOf(parent))
 	require.Error(t, apistatus.ObjectAlreadyRemoved{})
```
[the remark this answers is not in the document]
Comment (End-rey) [reply pkg/local_object_storage/metabase/delete_test.go:68]: I asked in #2943, I'll duplicate it here.

I added `err` to `require.ErrorIs` and everything all right.
https://github.com/nspcc-dev/neofs-node/blob/b135a0efe9c6e38ce69f53f9dd1c191d62f218bd/pkg/local_object_storage/metabase/delete_test.go#L63-L65
But there is `err == nil` and `ok` is false, so I don't know what to do. According to the comment above, both of them need to be removed.
https://github.com/nspcc-dev/neofs-node/blob/b135a0efe9c6e38ce69f53f9dd1c191d62f218bd/pkg/local_object_storage/metabase/delete_test.go#L67-L69

File: cmd/neofs-adm/internal/modules/morph/local_client.go [hunk 7]
```diff
@@ -212,13 +212,12 @@ func (l *localClient) CalculateNetworkFee(tx *transaction.Transaction) (int64, e
 			continue
 		}
 
-		fee, sizeDelta := fee.Calculate(ef, verificationScript)
-		netFee += fee
+		newFee, sizeDelta := fee.Calculate(ef, verificationScript)
```
Comment (carpawell) [inline cmd/neofs-adm/internal/modules/morph/local_client.go:215]: are you sure it is a suitable name?

Comment (roman-khimov) [reply cmd/neofs-adm/internal/modules/morph/local_client.go:215]: It's more of a `verificationFee`, yeah.

File: pkg/local_object_storage/metabase/delete_test.go (hunk 6 above)
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/local_object_storage/metabase/delete_test.go]: Looks like it was meant to end up with `apistatus.ObjectAlreadyRemoved{}` error (8107c8d1a9cc2635510988828e5bfeec13656b4d/#1318). @carpawell?

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/local_object_storage/metabase/delete_test.go (hunk 6 above)
[the remark this answers is not in the document]
Comment (carpawell) [reply pkg/local_object_storage/metabase/delete_test.go]: let us go this way: https://github.com/nspcc-dev/neofs-node/pull/2949

[the remark this answers is not in the document]
Comment (carpawell) [reply pkg/local_object_storage/metabase/delete_test.go]: and rebase this PR

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**